### PR TITLE
Sync anisgauss reg test to the other act line case

### PR DESCRIPTION
**Pull request by marchdf**
Increase epsilon a bit to not be equal to the grid size. This is an attempt to fix the dashboard for this case.

### CODE REVIEW

File: reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml [hunk 1]
```diff
@@ -131,9 +131,10 @@ realms:
         nacelle_cd: 1.0
         nacelle_area: 8.0
         air_density: 1.225
-        epsilon: [ 5.0, 2.0, 2.0 ]
+        epsilon: [ 5.0, 4.0, 4.0 ]
         epsilon_tower: 5.0
         turbine_base_pos: [ 0.0, 0.0, 0.0 ]
+        turbine_hub_pos: [ 0.0, 0.0, 90.0 ]
```
Comment (tasmith4) [inline reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml:137]: is this addition on purpose?  was the default not good, or are we just trying to match the other case to see what happens?

**Review by tasmith4 — APPROVED** [review]

**Review by mbkuhn — APPROVED** [review]

File: reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml (hunk 1 above)
Comment (marchdf) [reply reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml:137]: I think both your comments are going to the same thing. Yes I did need to make this change in #1320 to make things work well. I believe I had to do that so it matched the value in the openfast files? I can't remember...

[the remark this answers is not in the document]
Comment (marchdf) [reply reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml:137]: see above.

**Comment by marchdf** [comment]
Just to follow up. Looks like I fixed this case with this PR: https://my.cdash.org/tests/236645207
